config.toml

```diff
@@ -1,17 +1,17 @@
-base_url = "https://blog.codewars.com"
-title = "The Codewars Blog"
-description = ""
-default_language = "en"
-
-compile_sass = false
-highlight_code = true
-highlight_theme = "one-dark"
-build_search_index = false
-generate_feed = true
-feed_filename = "rss.xml"
-
-taxonomies = [
-  { name = "tags", paginate_by = 5, feed = true },
-]
-
-[extra]
+base_url = "https://blog.codewars.com"
+title = "The Codewars Blog"
+description = ""
+default_language = "en"
+
+compile_sass = false
+highlight_code = true

```

Comment:
How did you update upstream? It's reverting the change.

Reply:
Apparently, I updated it incorrectly :D
I got lost between GitHub Desktop, github console, and branches, forks, and merge conflicts. I have also no idea how to proceed :/ Sorry.


Maybe it would be easier to ditch this PR and create a new one. Oh dear, what a shame...

Reply:
Please see #14